pages/clubs-toolkit.jsx

```diff
@@ -26,8 +28,6 @@ var Toolkit = React.createClass({
   render: function() {
     return (
       <div>
-        <p>{this.props.header}</p>
```

Comment:
Haha, works for me. I was initially a little embarrassed when a simple change exposed how redundant my component was. Good to know it still has a use :)